common/src/main/java/io/quarkus/ts/openshift/common/test/BaseTest.java

```diff
@@ -0,0 +1,21 @@
+package io.quarkus.ts.openshift.common.test;
+
+import org.jboss.logging.Logger;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.TestInfo;
+
+public abstract class BaseTest {
+
+    private static final Logger LOG = Logger.getLogger(BaseTest.class.getName());
+
+    @BeforeEach
+    void setUp(TestInfo testInfo) {
+        LOG.debug(String.format("***** Test started: %s *****", testInfo.getDisplayName()));
+    }
+
+    @AfterEach
+    void tearDown(TestInfo testInfo) {
+        LOG.debug(String.format("***** Test finished: %s *****", testInfo.getDisplayName()));
+    }
+}
```

Comment:
Please don't do this, the JUnit extension is a better place for printing such info. And if I remember correctly, it's actually already there.

Reply:
I do want to avoid to annotate every single test case with a JUnit extension... 
Having a base test to cope with common features is a nice thing to have. Though we could move this logic into an extension and declare it in the base test instead. 

I could not see when the test starts and finishes, but will confirm this. If so, I'm fine to remove this.

Reply:
Ah this is for unit tests, not OpenShift tests. That makes sense now. The extension only applies to OpenShift tests. Feel free to take a look at how it's printing "running test xxx".

I feel like annotating a test class vs. making a test class extend a base class is the same amount of work, and the former generally offers more flexibility, so I'd recommend using that.

Reply:
Thanks for the pointer about the OpenShift tests. I've reported a new issue to refactor this, so we can log this information for both OpenShiftTests and QuarkusTests using the same JUnit extension: https://github.com/quarkus-qe/quarkus-openshift-test-suite/issues/113

Let be using a base class or a JUnit extension, It would require too many changes and it's out of scope for this PR (which is about to fix the Artemis tests). I will revert this change from this PR.